### ISSUE TITLE
WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.block)

### ISSUE THREAD

**Issue by VitthalBobade** [VitthalBobade/accessibilityinsightsreports#1]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=msftAI))
#### Target application
[BuddyConnect Admin Console | Login](https://buddyconnectwebdev.azurewebsites.net/)
#### Element path
.block
#### Snippet
`<button type="submit" name="Command" value="Login" class="btn btn-primary block full-width m-b">Login</button>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 4.38 (foreground color: #ffffff, background color: #ed1c24, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 84.0.4147.105
This accessibility issue was found using Accessibility Insights for Web 2.20.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
